plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/CloudSdkUtils.java

```diff
@@ -0,0 +1,124 @@
+/*******************************************************************************
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * All rights reserved. This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package com.google.cloud.tools.eclipse.appengine.localserver;
+
+import org.apache.maven.model.Model;
+import org.apache.maven.model.Plugin;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.m2e.core.MavenPlugin;
+import org.eclipse.m2e.core.project.IMavenProjectFacade;
+import org.eclipse.wst.common.project.facet.core.IFacetedProject;
+import org.eclipse.wst.common.project.facet.core.IProjectFacet;
+import org.eclipse.wst.common.project.facet.core.ProjectFacetsManager;
+import org.eclipse.wst.server.core.IRuntime;
+import org.eclipse.wst.server.core.internal.facets.FacetUtil;
+
+import com.google.cloud.tools.eclipse.util.MavenUtils;
+
+import java.util.List;
+
+/**
+ * Utility methods for Google Cloud SDK.
+ */
+@SuppressWarnings("restriction") // For FacetUtil
+public class CloudSdkUtils {
+  public static final String APPENGINE_GROUP_ID = "com.google.appengine";
```

Comment:
@elharo This code checks for the presence of the gcloud-maven-plugin, whose group ID is `com.google.appengine`.
On the other hand we'll need to extend this functionality to support the new Maven plugin if needed once it's ready to be used.